### ISSUE TITLE
Include examples that run in the browser

### ISSUE THREAD

**Issue by LegNeato** [Rust-GPU/rust-gpu.github.io#5]
We should have some demos / examples that run in the browser on the web.

**Comment by flying-sheep** [comment]
I came across this, maybe it has some boilerplate that would make writing such an example easier?

https://github.com/RustyBamboo/hash-shader

**Comment by LegNeato** [comment]
Yeah, I the blog post code uses wgpu as well. Someone needs to figure out all the right invocations to build everything for the web and then embed...all the dependencies support it. But it is a bit weird as it won't be running spirv, it will be running spirv translated to wgsl. Good enough for a demo though!

**Comment by schell** [comment]
I can contribute a glTF viewer from my project :) 

It probably needs just a little tweaking first.
